src/main/java/ca/wolfram/beta/symath/operations/PowerNode.java

```diff
@@ -8,15 +8,16 @@
 import java.util.Arrays;
 import java.util.List;
 
-public class PowerNode extends OperationNode{
+public class PowerNode extends OperationNode {
 
     private PowerNode(List<MathNode> children) {
         super(children);
     }
 
     /**
      * Index 0 is the base and Index 1 is the power.
-     * @throws RuntimeException if number of nodes is incorrect.
+     *
+     * @throws IllegalArgumentException if number of nodes is incorrect.
```

Comment:
but this documentation is about the scope of this one function. which depends on the scope of the functions it uses. In which case we should definitely have the definition here. And if we are having the definition here, i feel like we can just note down the number of arguements